test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -54,6 +54,64 @@ public void It_publishes_portable_apps_to_the_publish_folder_and_the_app_should_
         }
 
         [Fact]
+        public void It_runs_portable_apps_from_a_different_path_after_building()
```

Comment:
We shouldn't be running dotnet test3 tests in the SDK.

These tests belong in the CLI.